biz.aQute.launcher/src/aQute/launcher/Launcher.java

```diff
@@ -438,7 +451,7 @@ private int start(BundleContext systemContext, int result, BundleActivator activ
 	 * 
 	 * @param begin
 	 */
-	void update(long before) throws Exception {
+	List<Bundle> update(long before) throws Exception {
```

Comment:
Calling this method `update` is confusing. It should be called `start` since it is all about starting the specified list of bundle which may have already been updated.